templates/show_params.mako

```diff
@@ -205,8 +202,51 @@
     ${ render_msg( 'One or more of your original parameters may no longer be valid or displayed properly.', status='warning' ) }
 %endif
 
+
+<h3>Inheritance Chain</h3>
+<div class="inherit" style="background-color: #fff; font-weight:bold;">${hda.name | h}</div>
+
+% for dep in inherit_chain:
+    <div style="font-size: 36px; text-align: center; position: relative; top: 3px">&uarr;</div>
+    <div class="inherit">
+        '${dep[0].name | h}' in ${dep[1]}<br/>
+    </div>
+% endfor
+
+
+
+<h3>Command Line</h3>
+%if job and job.command_line and trans.user_is_admin():
+<pre style="white-space: pre-wrap; background: #1d1f21; color: white; padding: 1em;">
+${ job.command_line | h }</pre>
+%endif
+
+<style type="text/css">
+table.info_data_table {
+    table-layout: fixed;
+    word-break: break-word;
+}
+table.info_data_table td:nth-child(1) {
+    width: 25%;
+}
+</style>
+
+%if job and trans.user_is_admin():
+<h3>Job Metrics</h3>
+<table class="tabletip info_data_table">
+    <tbody>
+        <% job_metrics = trans.app.job_metrics %>
+        %for metric in sorted(job.metrics, key=lambda x:x.metric_name):
+            <% metric_title, metric_value = job_metrics.format( metric.plugin, metric.metric_name, metric.metric_value ) %>
+            <% metric_title = metric_title.replace(' (runtime environment variable)', '') %>
```

Comment:
I'm not on-board for this change I don't think. Even if I agree this shouldn't be appended to the environment variable - we need to drop it in the metric formatter since its only job is to produce the title that is presented here.